src/app/docs/flyout/flyout-docs-flyout.component.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'app-flyout-docs-flyout',
+  template: `
+    <div
+      class="sky-padding-even-large"
+    >
+      <h2>
+        Sample flyout
+      </h2>
+      <p>
+        Flyouts can display large quantities of supplementary information related to a task, including:
+      </p>
+        <ul>
+          <li><a href="https://developer.blackbaud.com/skyux/components/list">lists</a></li>
```

Comment:
```suggestion
          <li>lists</li>
```
Seems like we should get rid of this link and avoid steering users toward that module given that we want to de-emphasize the module in preference for data entry grid or other options.

Reply:
Yep, good idea.